src/utils/daos_autotest.c

```diff
@@ -760,11 +834,14 @@ struct step {
 };
 
 static struct step steps[] = {
+	/** Pre-test checks */
+	{ 0,	"Dumping AttachInfo",		attachinfo,	100 },
```

Comment:
This should verify that the agent is correctly running instead.

Reply:
This is addressed now in the daos command itself. 
When daos_agent isn't running it displays a meaningful message with a new errno:
```
daos ERR  src/common/drpc.c:217 unixcomm_connect() Failed to connect to /var/run/daos_agent/daos_agent.sock, errno=2(No such file or directory)
mgmt ERR  src/mgmt/cli_mgmt.c:234 get_attach_info() failed to connect to /var/run/daos_agent/daos_agent.sock DER_AGENT_COMM(-2034): 'Agent communication error'
ERROR: daos: daos_init() failed: DER_AGENT_COMM(-2034): Agent communication error
```